pkg/guacanalytics/patchPlanning.go

```diff
@@ -104,6 +104,17 @@ func SearchDependenciesFromStartNode(ctx context.Context, gqlClient graphql.Clie
 		q.queue = append(q.queue, startID)
 	}
 
+	err = q.bfsOfDependencies(ctx, gqlClient, stopID, maxDepth)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return q.nodeMap, path, nil
+
+}
+
+// bfsOfDependencies performs a breadth-first search on a graph to find dependencies
```

Comment:
I think it is technically bfsOfDependents?

Reply:
Because it finds the dependents i.e. which packages/artifact/sources are using it.

Reply:
Maybe you should rename the search function SearchDependentsFromStartNode?  What do you think @lumjjb ?

Reply:
Good idea, I have done that.

Reply:
I don’t think that it needs to be called `SearchDependentsFromStartNode` because bfs specifies what type of search we are running.

Reply:
Specifically the bfsOfDependencies should be bfsOfDependents.  In terms of the search change, I am fixing it here #1142.

Reply:
@rmetzman, Can this comment be resolved?